Makefile.checker

```diff
@@ -34,6 +34,20 @@ CHECKMLLIBFILE := checker/.mllibfiles
 CHECKERDEPS := $(addsuffix .d, $(CHECKMLDFILE) $(CHECKMLLIBFILE))
 -include $(CHECKERDEPS)
 
+# Copied files
+checker/esubst.mli: kernel/esubst.mli
+	cp -a $< $@
+	sed -i '1i(* AUTOGENERATED FILE: DO NOT EDIT *)\n\n\n\n\n\n\n\n' $@
+checker/esubst.ml: kernel/esubst.ml
+	cp -a $< $@
+	sed -i '1i(* AUTOGENERATED FILE: DO NOT EDIT *)\n\n\n\n\n\n\n\n' $@
+checker/names.mli: kernel/names.mli
+	cp -a $< $@
+	sed -i '1i(* AUTOGENERATED FILE: DO NOT EDIT *)\n\n\n\n\n\n\n\n' $@
+checker/names.ml: kernel/names.ml
+	cp -a $< $@
+	sed -i '1i(* AUTOGENERATED FILE: DO NOT EDIT *)\n\n\n\n\n\n\n\n' $@
```

Comment:
IIRC `sed -i` is not ok on OSX (for sure not in busybox).
You could `( echo "DO NOT" ;  cat file1 ) > file2`

Reply:
It seems that it passed the MacOS tests though. But let's avoid breaking compatibility with BusyBox again if we can.

Reply:
That's strange, AFAICT `sed -i` is OK in all platforms, including Busybox https://busybox.net/downloads/BusyBox.html#sed